build_scripts/build_runner.py

```diff
@@ -720,7 +720,7 @@ def main():
                         choices=['release', 'debug'],
                         help='Type of build')
     parser.add_argument('-p', "--product-type", default='linux',
-                        choices=['linux', 'embedded', 'open_source', 'windows'],
```

Comment:
Don't remove 'open_source'. It's needed for closed open source builds)

Reply:
Ok. Strange that it was deleted...

Reply:
Approve pls